pkg/controllers/openshift-route-controller-manager.go

```diff
@@ -116,7 +116,7 @@ func (s *OCPRouteControllerManager) Run(ctx context.Context, ready chan<- struct
 		return err
 	}
 
-	if err := route_controller_manager.RunRouteControllerManager(s.config, clientConfig); err != nil {
+	if err := route_controller_manager.RunRouteControllerManager(s.config, clientConfig, ctx); err != nil {
 		return err
```

Comment:
this can be simplified further as follows: https://github.com/openshift/microshift/pull/1033/commits/9866670a71bd4eb005c58b09aef56357db82689f because the call is blocking now

Reply:
done